src/Navier_Stokes_equations_routines.f90

```diff
@@ -11870,7 +11870,7 @@ SUBROUTINE NavierStokes_Couple1D0D(controlLoop,solver,err,error,*)
         !allocate array for mpi communication
         ALLOCATE(globalConverged(numberOfComputationalNodes),STAT=ERR) 
         IF(ERR/=0) CALL FlagError("Could not allocate global convergence check array.",ERR,ERROR,*999)
-        CALL MPI_ALLGATHER(localConverged,1,MPI_LOGICAL,globalConverged,1,MPI_LOGICAL, &
+        CALL MPI_ALLGATHER(localConverged,1,MPI_INTEGER,globalConverged,1,MPI_INTEGER, &
          & COMPUTATIONAL_ENVIRONMENT%MPI_COMM,MPI_IERROR)
```

Comment:
What is the reason why this has been changed from logical to integer? The array is logical and so it should be MPI_LOGICAL? Also, why does this affect python? It is an internal routine? Finally, shouldn't the changes be committed to cm and then pulled through?